pom.xml

```diff
@@ -40,6 +40,21 @@
 			<artifactId>springdoc-openapi-ui</artifactId>
 			<version>1.5.2</version>
 		</dependency>
+
+		<dependency>
```

Comment:
The dependency aren't in Stage ? Please check it

Reply:
what do you mean exactly?